### PR TITLE
fix hang in FindRuntimeSpecs

### PR DESCRIPTION
**Pull request by anurse**
Because `imports` was outside the loop, in the case where the last runtime in the list had no imports but the penultimate one DID have imports, the loop became infinite (technically it bounced between two call stack levels infinitely, it didn't actually get stuck in a single infinite loop)

/cc @lodejard @davidfowl

### CODE REVIEW

**Comment by davidfowl** [comment]
:shipit:

File: src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs [hunk 1]
```diff
@@ -555,9 +555,9 @@ public RestoreCommand(FrameworkName fallbackFramework)
         {
             effectiveRuntimeSpecs.RemoveAll(spec => spec.Name == runtimeName);
 
-            IEnumerable<string> imports = null;
             foreach (var runtimeFile in runtimeFiles)
             {
+                IEnumerable<string> imports = null;
```
Comment (ericstj) [inline src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs]: It looks like the subsequent `if (imports != null)` check will always be false.  Are you sure that the fix isn't instead just to close the `foreach` statement before the following `if (imports != null)` check?

Comment (anurse) [inline src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs]: Indeed it should! Fixed now!

**Comment by ericstj** [comment]
works for me :+1:

**Comment by ericstj** [comment]
I just noticed that we still hit this bug in beta5, however it is fixed in beta6.  Was that intentional?

**Pull request closed without merging**